helm/values.yaml

```diff
@@ -202,14 +202,25 @@ fluentbit:
     service: ""
     inputs: |
       [INPUT]
-          Name tail
-          Path /tmp/logs/*.log
-          Tag app.access.log
-          Mem_Buf_Limit 5MB
+          Name            tail
+          Path            /tmp/logs/*.log
```

Comment:
why all the whitespace changes?

Reply:
I thought it would be easier to read if the values were aligned with each other.